src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -52,6 +74,9 @@ const defaultProps = {
     value: undefined,
     defaultValue: undefined,
     forceActiveLabel: false,
+    shouldSaveDraft: false,
```

Comment:
NAB Aren't Forms supposed to [save draft values by default](https://docs.google.com/document/d/1DI4jQ0nyfi2JczhPT440orwlXsTP_O-aGvg-HLmcbwI/edit#heading=h.um36291aebnb)?

Reply:
Thanks for the review @neil-marcellini! We decided to default draft saving to false so it’s intentional and we avoid saving sensitive information.